style.css

```diff
@@ -34,10 +33,100 @@ body {
   text-decoration: none;
 }
 
+.toggleNav {
+  width: 18px;
+  height: 12px;
+  align-self: center;
+}
+
+.toggleNav .exit {
+  display: none;
+}
+
+.show .blur {
+  position: fixed;
+  z-index: 1;
+  width: 100vw;
+  height: 100vh;
+  backdrop-filter: blur(5px);
+}
+
 .menu > ul {
   display: none;
 }
 
+.show {
+  display: block !important;
+}
+
+.line > div {
+  width: 134px;
+  height: 5px;
+  border-radius: 10px;
+  background-color: #0d151e;
+  margin-top: 32px;
+  margin-bottom: 8px;
+}
+
+.show .menu ul {
+  width: 100vw;
+  display: block;
+  justify-content: flex-end;
+  list-style: none;
+  gap: 20px;
+  color: #344563;
+  position: absolute;
+  height: 40px;
+  left: 10px;
+}
+
+.show .toggleNav {
+  display: none;
+}
+
+.show .menu > div {
+  display: block;
+}
+
+.show .exit {
+  width: 20px;
+  height: 20px;
+  position: absolute;
+  display: block;
+  right: 20px;
+  top: -20px;
+}
+
+.show .menu {
+  width: 100vw;
+  position: relative;
+  display: block;
+  z-index: 9;
+  height: 40px;
+  top: 60px;
+}
+
+.show .menu a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.show .menu li {
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  padding-bottom: 15px;
+}
+
+.show .mobile-overlay {
+  background-color: #6070ffe3 !important;
+  height: 100vh;
+  width: 100vw;
+  position: fixed;
+  z-index: 4;
+}
```

Comment:
- [ ] You are currently using a different color `#6070ffe3` instead of the `#6070ff` from the design which makes your design slightly different from the original one as shown below :point_down: . Kindly ensure that you the hamburger button and menu follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins).  
- [ ] Also, kindly ensure that the overlay stretches from top to bottom and not just part  of the page to match the design. :wink: 

You can consider using the same color and adding `mix-blend-mode: multiply` to achieve the same opacity and overlay design as the figma. :wink: 


### Your design 

![Screenshot from 2022-12-19 18-00-07](https://user-images.githubusercontent.com/65233860/208490262-7b7fccb1-7efe-4fda-8925-77804cfa0902.png)


### Figma design 


![Screenshot from 2022-12-19 17-50-46](https://user-images.githubusercontent.com/65233860/208489535-632728cc-53da-4e23-8a45-50a51721d134.png)


![Screenshot from 2022-12-19 18-02-00](https://user-images.githubusercontent.com/65233860/208490446-c1adfc20-fedc-42c9-849c-7bf86dcd3d80.png)

Reply:
well resolved! ✔️